src/Azure.Functions.Cli/Helpers/VersionHelper.cs

```diff
@@ -184,6 +184,28 @@ public string CoreToolsReleaseNumber
                 }
             }
             public CoreToolsRelease ReleaseDetail { get; set; }
+            public string CoreToolsReleaseZipfile
```

Comment:
are there any tests you can add to this as well? I would take a fake CDN link with the same format and then make sure that parsing it returns the expected value

Reply:
@aishwaryabh added test cases for the changes.